Project.toml

```diff
@@ -1,21 +1,14 @@
 name = "ITensorTDVP"
 uuid = "25707e16-a4db-4a07-99d9-4d67b7af0342"
 authors = ["Matthew Fishman <mfishman@flatironinstitute.org> and contributors"]
-version = "0.2.1"
+version = "0.3"
```

Comment:
I'm marking this as a breaking change since it is such a big change to the codebase and at this point users may have been relying on internals (especially because of the old example `05_tdvp_nonuniform_timesteps.jl`).

Reply:
Good call